manifests/params.pp

```diff
@@ -94,4 +94,5 @@
   $environment_variables      = {}
   $config_variables           = {}
   $config_kernel_variables    = {}
+  $ulimit                     = 4096
```

Comment:
RabbitMQ team member here.

The default is `1024` on most distributions and I suspect our docs use it as an example value. `4096`, `16384`, or even `65536` are sensible values since `ulimit -n` is scoped per OS user.

I'd recommend `16384`.